Add MoonGuard plugin

guetteman opened the pull request.

danharrin commented on content/plugins/moonguard.md:
Please add your author slug to the start of the file names, and the `slug` here, like all the other plugins

guetteman replied:
Done!

awcodes commented on content/plugins/luis-guette-moonguard.md:
```diff
@@ -0,0 +1,15 @@
+categories: ['analytics', 'developer-tool']
+description: Plugin that providers utilities to monitor your app status and performance once it is released to production.
+discord_url: https://discord.gg/vCCy4aJxnY
+docs_url: https://docs.moonguard.dev
```
Please change this to 'url' instead of 'docs_url' since it is not a markdown file.

guetteman replied:
done!

saade commented:
Hey! Your plugin's `README` is loaded into the plugin page on the Filament website. There is now a duplicate image on your page, as it also exists in your documentation. You can remove it from the Filament website by giving it a `filament-hidden` class, so then it is only shown once.

Please let us know when you've addressed these issues.

guetteman commented:
> Hey! Your plugin's `README` is loaded into the plugin page on the Filament website. There is now a duplicate image on your page, as it also exists in your documentation. You can remove it from the Filament website by giving it a `filament-hidden` class, so then it is only shown once.
> 
> Please let us know when you've addressed these issues.

Done
